### DISCUSSION TITLE
How to enable native-like warnings from `dart-sass` in `sass-loader`?

### DISCUSSION THREAD

**Discussion by Andrew15-5** [webpack/webpack#17604] [Q&A]
I have this in my `webpack.config.js`:

```js
...
    rules: [
      {
        test: /\.s[ac]ss$/i,
        exclude: '/node_modules/',
        use: [
          // 'style-loader',
          MiniCssExtractPlugin.loader,
          {
            loader: 'css-loader',
            options: { url: false },
          },
          {
            loader: 'sass-loader',
            options: {
              implementation: require('sass'),
            },
          },
        ],
      },
    ]
...
```

Even though `sass-loader` uses `dart-sass`, I can't see the same `dart-sass` style warnings:

```text
Warning on line 5, column 1 of src/styles/dir/_partial.sass:
This selector doesn't have any properties and won't be rendered.
  ╷
5 │ asdf
  │ ^^^^
  ╵
```

**If** it shows a warning, then this is what I get (some warnings are ignored):

```text
This selector doesn't have any properties and won't be rendered.

null
 @ ./src/styles/main.sass
```

I simply can't even know where is the warning coming from.

I tried getting an error, and it is pretty much 1:1 but without pretty colors. So errors are cool, but warnings are different for some reason. I use the same JS package directly and through `sass-loader` + `webpack --watch`.

P.S. Link to original question on Stack Overflow: https://stackoverflow.com/q/76907996/16722274.

**Comment by alexander-akait** [comment]
@Andrew15-5 Sorry for delay, can you open an issue in `sass-loader`?

**Reply by Andrew15-5** [reply]
I can, but I'm not involved with sass or webpack at the moment, so it's not a problem for current me. I'm very busy, too, so I don't have time or desire in getting this issue resolved, you know?
